msteams-platform/resources/teams-updates.md

```diff
@@ -0,0 +1,55 @@
+
+## Known issues
+
+> [!NOTE]
+> It's recommended to test apps, tabs, messaging extensions, bots, and link unfurling after switching from the Classic Teams Client to the new Teams client.
+
+* You can't install, uninstall, pin, and unpin apps in the new Teams client. It's recommended to perform these actions in the Classic Teams Client and the changes are reflected in the New Teams Client. For more information, see [Pin apps in Microsoft Teams](https://support.microsoft.com/office/pin-an-app-for-easy-access-3045fd44-6604-4ba7-8ecc-1c0d525e89ec).
+
+* You can't add, update, rename, and remove tabs from chats and channels in the new Teams client. It's recommended to perform these actions in the Classic Teams Client and the changes are reflected in the New Teams Client. For more information, see [Use an app in a tab in a channel or chat](https://support.microsoft.com/office/use-an-app-in-a-tab-in-a-channel-or-chat-83d0514f-2134-4db5-80f2-e9b43e111d57).
+
+* [Share in Teams](../concepts/build-and-test/share-to-teams-from-personal-app-or-tab.md) isn't supported in the new Teams client.
+
+* APIs used in Teams that require [device permissions](../concepts/device-capabilities/native-device-permissions.md) aren't supported.
+
+* [Adaptive card tabs](../tabs/how-to/build-adaptive-card-tabs.md) aren't supported in the new Teams client.
+
+For more information on known issues and gaps in the New Teams Client, see [New Microsoft Teams](/microsoftteams/new-teams-desktop-admin?tabs=teams-admin-center#known-issues).
+
+If your app is working fine in the Classic Teams Client but has issues in the new Teams then raise an issue on [this page](https://github.com/MicrosoftDocs/msteams-docs/issues/new?title=&body=%F0%9F%9A%A8%20Looks%20like%20you%20arrived%20from%20the%20internal%20review%20site%20%0A%F0%9F%9A%A8%20Do%20not%20enter%20Microsoft%20confidential%20information%20here%0A%0A%5BEnter%20feedback%20here%5D%0A%0A%0A---%0A%23%23%23%23%20Document%20Details%0A%0A%E2%9A%A0%20*Do%20not%20edit%20this%20section.%20It%20is%20required%20for%20learn.microsoft.com%20%E2%9E%9F%20GitHub%20issue%20linking.*%0A%0A*%20ID%3A%204c405ed3-f304-ff58-d32d-108fc725bb6a%0A*%20Version%20Independent%20ID%3A%204c405ed3-f304-ff58-d32d-108fc725bb6a%0A*%20Content%3A%20%5BMicrosoft%20Teams%20updates%20-%20Teams%5D(https%3A%2F%2Freview.learn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fresources%2Fteams-updates%3Fbranch%3Dpr-en-us-8713)%0A*%20Content%20Source%3A%20%5Bmsteams-platform%2Fresources%2Fteams-updates.md%5D(https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fresources%2Fteams-updates.md)%0A*%20Service%3A%20**msteams**%0A*%20GitHub%20Login%3A%20%40v-ypalikila%0A*%20Microsoft%20Alias%3A%20**lajanuar**). For any other issues, request you to raise an issue on [Support and feedback](../feedback.md#developer-community-forums).
```

Comment:
This link doesn't seem correct. How was this generated?

Reply:
@Wajeed-msft ,

Can you please help us here. Is it possible to have a shorter link.

Reply:
@ydogandjiev - This link gets auto generated for each page for developer to raise feedback.
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/31851992/8d5e86a4-e83f-45bd-ba57-4fd9a9c49016)

This opens up a prefilled new issues page: 
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/31851992/7049f1ae-8253-4c69-887b-e93f4412a028)

@surbhigupta12 - could you please replace this link (review site) to the prod link?

Reply:
Hi @Wajeed-msft,

We've updated the link with Prod link "https://github.com/MicrosoftDocs/msteams-docs/issues/new?title=&body=%0A%0A%5BEnter%20feedback%20here%5D%0A%0A%0A---%0A%23%23%23%23%20Document%20Details%0A%0A%E2%9A%A0%20*Do%20not%20edit%20this%20section.%20It%20is%20required%20for%20learn.microsoft.com%20%E2%9E%9F%20GitHub%20issue%20linking.*%0A%0A*%20ID%3A%2019ddf42e-0a47-7717-52d4-e549155480a2%0A*%20Version%20Independent%20ID%3A%204bbe9beb-233f-cfd5-097b-f280aab5fde8%0A*%20Content%3A%20%5BMicrosoft%20Teams%20developer%20community%20support%20and%20feedback%20-%20Teams%5D(https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Ffeedback)%0A*%20Content%20Source%3A%20%5Bmsteams-platform%2Ffeedback.md%5D(https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Ffeedback.md)%0A*%20Service%3A%20**msteams**%0A*%20GitHub%20Login%3A%20%40surbhigupta%0A*%20Microsoft%20Alias%3A%20**lajanuar**".

Thanks,
Yash